scripts/apothecary/formulas/openSSL/buildwin.cmd

```diff
@@ -0,0 +1,46 @@
+@echo off
```

Comment:
It would probably have to work like the way fmod is handled since the windows binaries come as exe files so the compiled libs would have to be copied over from the install. 

It could be done in a shell script but that would still require using cmd //c "call etc... and I was having issues calling nmake because it is not on a default path and there were issues with having a space in the URL namely the Program Files folder. The batch file was just cleaner and less hassle plus there was an issue with which version of Perl was being used since I had installed active perl to the system path but git bash was using its own version